nbbuild/antsrc/org/netbeans/nbbuild/ModuleListParser.java

```diff
@@ -190,6 +190,9 @@ public static void resetCaches() {
                 registerTimestampAndSize(new File(root, "nbbuild" + File.separatorChar + "cluster.properties"), timestampsAndSizes);
                 registerTimestampAndSize(new File(root, "nbbuild" + File.separatorChar + "build.properties"), timestampsAndSizes);
                 registerTimestampAndSize(new File(root, "nbbuild" + File.separatorChar + "user.build.properties"), timestampsAndSizes);
+
+                doFastScan = false;
```

Comment:
This is not very professional, but disks are faster these days and computers have more memory to fit `inode` info for most of NetBeans sources into memory.

Maybe we don't have to optimize this anymore...